students/stachowska_agata/lesson_01_basics/01Lesson.py

```diff
@@ -0,0 +1,102 @@
+print("1.1 «Sum of three numbers»")
```

Comment:
Create every exercise in a separate file

Example:
students/stachowska_agata/lesson_01_basics/apple_sharing.py